src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -633,12 +644,155 @@ public Snapshot makeSnapshotFromSnapshotRequest(SnapshotRequestModel snapshotReq
+            .conceptRelationships(
+                List.of(
+                    "fpk_measurement_concept",
+                    "fpk_measurement_unit",
+                    "fpk_measurement_concept_s",
+                    "fpk_measurement_value",
+                    "fpk_measurement_type_concept",
+                    "fpk_measurement_operator")));
+    tableMap.put(
+        "Visit",
+        new SnapshotBuilderTable()
+            .datasetTableName("visit_occurrence")
+            .personRelationship("fpk_person_visit")
+            .conceptRelationships(
+                List.of(
+                    "fpk_visit_preceding",
+                    "fpk_visit_concept_s",
+                    "fpk_visit_type_concept",
+                    "fpk_visit_concept",
+                    "fpk_visit_discharge")));
+    tableMap.put(
+        "Device",
+        new SnapshotBuilderTable()
+            .datasetTableName("device_exposure")
+            .personRelationship("fpk_person_device")
+            .conceptRelationships(
+                List.of("fpk_device_concept", "fpk_device_concept_s", "fpk_device_type_concept")));
+    tableMap.put(
+        "Condition",
+        new SnapshotBuilderTable()
+            .datasetTableName("condition_occurrence")
+            .personRelationship("fpk_person_condition")
+            .conceptRelationships(
+                List.of(
+                    "fpk_condition_concept",
+                    "fpk_condition_type_concept",
+                    "fpk_condition_status_concept",
+                    "fpk_condition_concept_s")));
+    tableMap.put(
+        "Procedure",
+        new SnapshotBuilderTable()
+            .datasetTableName("procedure_occurrence")
+            .personRelationship("fpk_person_procedure")
+            .conceptRelationships(
+                List.of(
+                    "fpk_procedure_concept",
+                    "fpk_procedure_concept_s",
+                    "fpk_procedure_type_concept",
+                    "fpk_procedure_modifier")));
+    tableMap.put(
+        "Observation",
+        new SnapshotBuilderTable()
+            .datasetTableName("observation")
+            .personRelationship("fpk_person_observation")
+            .conceptRelationships(
+                List.of(
+                    "fpk_observation_concept",
+                    "fpk_observation_concept_s",
+                    "fpk_observation_unit",
+                    "fpk_observation_qualifier",
+                    "fpk_observation_type_concept",
+                    "fpk_observation_value")));
+    tableMap.put("Year of Birth", new SnapshotBuilderTable().datasetTableName("person"));
+    tableMap.put(
+        "Ethnicity",
+        new SnapshotBuilderTable()
+            .datasetTableName("person")
+            .conceptRelationships(
+                List.of("fpk_person_ethnicity_concept", "fpk_person_ethnicity_concept_s")));
+    tableMap.put(
+        "Race",
+        new SnapshotBuilderTable()
```

Comment:
Why are year of birth, ethnicity, and race in the tableMap?

Reply:
Good point -I can remove them. These relationships will now be covered by the demographic option.